FlatMate/View/SignupView.swift

```diff
@@ -119,8 +124,8 @@ struct SignupView: View {
             return false
         }
         
-        if password.count < 12 {
-            errorMessage = "Password must be at least 12 characters long."
+        if password.count < 8 {
+            errorMessage = "Password must be at least 8 characters long."
```

Comment:
Do we validate this change on Firebase's end? If not we're good to merge

Reply:
It looks like we only validate on the client-side before sending any data to Firebase, so we should be good to merge!